Source/Doxygen/pydoc.cxx

```diff
@@ -488,11 +498,16 @@ void PyDocConverter::handleCode(DoxygenEntity &tag, std::string &translatedComme
   translatedComment += ".. code-block:: c++\n\n";
 
   // For now on, use extra indent level for all the subsequent lines.
-  codeIndent += m_indent;
+  codeIndent = m_indent;
```

Comment:
Are you sure about this change? It looks logical to me to add to the existing indent, i.e. I don't see why was the old code wrong. The commit message speaks about a new unit test showing, but it doesn't seem to have been added yet?

And if this is really correct, the comment above needs to be updated after this change, it's not an "extra" indent level any longer. In fact, the whole `codeIndent` variable should be just removed if we really can just use `m_indent` instead.

Reply:
Thanks for the review.  As far as I can tell this change is correct.  The indentation handling in this code is a little bit confusing, and took me some time (and lots of printf debugging) to wrap my head around as I'm not the original author.  Above the modified line in question, `codeIntent` stores the indentation necessary for adding text to `translatedComments`.  With the standard doxygen style represented in the test cases, this would be an empty string.  At the line in question the current value of `m_indent` is set as 4 spaces by the indent guard.  Those spaces are added so that the translated comments in the code block are indented by that amount.  So with the standard doxygen comment style, the change that I introduced would have no effect.

However, the problem is that with an alternative doxygen style (see below), there is a space in front of everything that was handled by the parser.  This extra space shows up in the variable `codeIndent`, and it is needed when directly adding a new line of text to `translatedComment`.  However, I don't believe it is correct to then use that extra space plus the 4 indentations in front of the translated content, because the translated content itself already contains that extra space in front.  This results in 5 space indentation within code blocks when using the alternative comment style.

Having said that, it might be worth adding some additional comments to the code for better clarification.  What workflows are typically used to update pull requests?  In this particular case a new commit could make sense, but in other cases is a rebase and force push to the branch used?

I did attempt to add a new test case in 91a90b8d2, but it's my first time so I certainly may have missed something!  The new case is doxygen_basic_translate_style2.i, and I updated common.mk.

Here is what the alternative comment style looks like:
<pre>
/** Line 1
 *  Line 2
 */
</pre>
This results in an extra leading space in content read by the parser.  It seems to be mostly already handled correctly by the translator, but as I mentioned indentation within a code block was not correct.  And this comment style was not previously being tested.  (By the way, is it preferable to commit the new test case before the fix, so that the failure to achieve the intended behavior can be more clearly demonstrated?)

Reply:
> 
> 
> The commit message speaks about a new unit test showing, but it doesn't seem to have been added yet?

I just pushed an update 64b2113e6 because there was an oversight in the original commit of the new test case, doxygen_basic_translate_style2.  The python _runme.py file with expected results was referencing an out of date module name that I had since changed.  This may explain questions about the new unit test, which would have failed (was passing on my local machine due to old files hanging around...)

With regards to my earlier question about how to make updates to the PR: I'm happy to rebase this (and some of the other suggested cleanup I'm working on) and force push the updates to clean up the history if that is preferred to new commits.